docs/components/toasts/index.html

```diff
@@ -0,0 +1,49 @@
+---
+title: Toasts
+also:
+  elements/hx-toast: <hx-toast>
+---
+{% extends 'component.njk' %}
+{% block content %}
+  <section>
+    <p>
+      The toast component confirms a request, provides a short informational message, 
+      or lets the user know that there's an error with the requested action. 
+    </p>
+  </section>
+
+  <section>
+    <h2 id="demo">Demo</h2>
```

Comment:
Make a note here about the advanced functionality (e.g., location, stacking, animation, accessibility) coming in a later PR.